common/app/views/fragments/submeta.scala.html

```diff
@@ -28,15 +29,48 @@ <h2 class="submeta__head">Topics</h2>
                     @fragments.social(content)
                 </div>
                 <div class="js-save-for-later submeta__save-for-later" data-position="bottom"></div>
+                @if(SyndicationReprintEnabledSwitch.isSwitchedOn) {
+                    @if(content.isSyndicatable) {
+                        <div class="submeta__syndication">
+                            @fragments.syndication(content)
+                        </div>
+                    }
+                }
```

Comment:
I think it might be worth putting the two `if` conditions in the syndication fragment. It does mean the fragment gets pull in a few times but it will reduce the repetition of code.